corehq/apps/locations/models.py

```diff
@@ -543,6 +544,7 @@ def location_type(self):
         return self.location_type_object.name
 
     _sql_location_type = None
+
     @location_type.setter
     def location_type(self, value):
```

Comment:
Agree that seems like a reasonable time to violate this rule.  I think after having corrected the actual mistakes, dimagimon + people's judgment should be sufficient to not have to correct this error en-masse again.